src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/ResponseOptions/ResponseOptions.utils.tsx

```diff
@@ -58,13 +62,75 @@ export const getResponseItem = ({
     />
   );
 
+  const renderSingleSelectionPerRow = () => {
+    const updatedOptions = activityItem?.responseValues.options.map(({ id, text }, index) => ({
```

Comment:
Maybe we can merge updatedOptions and optionValueMapper into one mapper, as they both map the same value (activityItem?.responseValues.options).